src/olympia/bandwagon/views.py

```diff
@@ -84,7 +84,13 @@ def legacy_redirect(request, uuid, edit=False):
     collection = get_object_or_404(Collection.objects, **{key: uuid})
     if edit:
         return http.HttpResponseRedirect(collection.edit_url())
-    to = collection.get_url_path() + '?' + request.GET.urlencode()
+
+    to = collection.get_url_path()
+    params = request.GET.urlencode()
+
+    if params:
+        to += '?' + params
```

Comment:
This apparently only broke during Django 1.11 but I think this can be uplifted too

Reply:
decided to not uplift that… too small and really just spammy and legacy redirects anyway